content/workflows/fairness-reweighing-preprocessor.md

```diff
@@ -0,0 +1,12 @@
++++
+title= "Reweighing as a preprocessor"
+images =  ["/workflows/images/fairness-reweighing-preprocessor.png"]
+type = "workflows"
+blog_link =  "/blog/2023/2023-08-25-fairness-reweighing-preprocessor/"
+video = ""
+download = "fairness-reweighing-preprocessor.ows"
+workflows = ["Fairness"]
+weight = 1002
++++
+
+We can use the reweighing fairness algorithm for more than just adding weights to a dataset. It can also be used as a preprocessor for a specific model. This workflow illustrates how to use the Reweighing widget as a preprocessor for the Logistic Regression model. Initially, we load the dataset and input it into the Test and Score widget, which we will use to evaluate our model. Now, we need to connect a Logistic Regression model, which has the Reweighing widget as a preprocessor, to the Test and Score widget. Doing so ensures our data gets reweighed before the model learns from it. We then visualize the results using a Box Plot. For a comprehensive understanding, we compare these results against those derived from a Logistic Regression model that does not use the Reweighing widget.
```

Comment:
"For a comprehensive understanding" -> remove

"we compare these results against those derived from a Logistic Regression model that does not use the Reweighing widget." -> "We compare results to logistic regression without reweighing."

I'd skip the box plot sentence.